runtime/test/strategies/resolve-recipe-test.js

```diff
@@ -40,7 +40,8 @@ describe('resolve recipe', function() {
     let [recipe] = manifest.recipes;
     assert.isTrue(recipe.normalize());
 
-    await noResult(arc, ResolveRecipe, recipe);
+    recipe = await onlyResult(arc, ResolveRecipe, recipe);
```

Comment:
This doesn't seem right?

Reply:
Yup, this shouldn't be here, reverted this file :)